tests/vocabulary.rs

```diff
@@ -0,0 +1,269 @@
+// Copyright 2016 Mozilla
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+// this file except in compliance with the License. You may obtain a copy of the
+// License at http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software distributed
+// under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+// CONDITIONS OF ANY KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations under the License.
+
+#[macro_use]
+extern crate lazy_static;
+
+extern crate mentat;
+extern crate mentat_core;
+extern crate mentat_db;
+extern crate rusqlite;
+
+use mentat::vocabulary;
+
+use mentat::vocabulary::{
+    VersionedStore,
+    VocabularyCheck,
+    VocabularyOutcome,
+};
+
+use mentat::query::IntoResult;
+
+use mentat_core::{
+    HasSchema,
+};
+
+use mentat::{
+    Conn,
+    NamespacedKeyword,
+    Queryable,
+    TypedValue,
+    ValueType,
+};
+
+use mentat::entity_builder::BuildTerms;
+
+use mentat::errors::{
+    Error,
+    ErrorKind,
+};
+
+lazy_static! {
```

Comment:
See earlier comment about `core/values.rs` -- we have a macro for lazy static ns keywords :)

Reply:
Aye, but it produces `edn::Value::NamespacedKeyword`.